packages/pf3-component-mapper/src/form-fields/date-time-picker/date-time-picker.js

```diff
@@ -45,6 +45,18 @@ export class DateTimePicker extends React.Component {
     });
   }
 
+  componentDidUpdate({ pristine }) {
+    if (this.props.pristine === true && pristine === false) {
+      const date = this.props.value ? typeof this.props.value === 'string' ? new Date(this.props.value) : this.props.value : undefined;
```

Comment:
this is used at two places now, I would extract this function to make additional modifications easier.